SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetConnection.cpp

```diff
@@ -153,14 +153,19 @@ void USpatialNetConnection::InitHeartbeat(FTimerManager* InTimerManager, Worker_
 
 void USpatialNetConnection::SetHeartbeatTimeoutTimer()
 {
+	float Timeout = GetDefault<USpatialGDKSettings>()->HeartbeatTimeoutSeconds;
+#if WITH_EDITOR
```

Comment:
This is going to happen even if you launch an editor build as a standalone, which might not always be desirable, e.g. when you want to test the gameplay logic surrounding players timing out.

Reply:
As per Alex's suggestion, I'll make this a configurable setting, so users can modify it if the circumstances require. Acceptable?